### DISCUSSION TITLE
Fatal Scar Error

### DISCUSSION THREAD

**Discussion by Reak2382** [Woprok/AOE4-AdvancedGameSettings#116] [General]
Having an Error since the latest patch causing the entire game to become blue unless paused (which isnt possible in multiplayer).
I really appreciate your work on the mod and hope that you could fix it to be working again since theres nothing comparing to this and would love to see it come back for the DLC and future patches.

Thanks for hearing me out)

**Comment by Woprok** [comment]
@Reak2382 
 see explanation https://github.com/Woprok/AOE4-AdvancedGameSettings/discussions/117
